### PR TITLE
Filtering fixes for grids

### PR DESCRIPTION
**Pull request by mpisacic**

### CODE REVIEW

File: src/components/QuickGrid/QuickGrid.Props.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import * as React from 'react';
 import { ITooltipProps } from '../Tooltip/Tooltip.props';
 
+export interface IQuickGrid {
```
Comment (vinkobedek) [inline src/components/QuickGrid/QuickGrid.Props.ts]: why is it showing this here? this was a part of some other pull request?

File: src/components/QuickGrid/DataSelectors.ts [hunk 2]
```diff
@@ -64,19 +75,19 @@ const sortRows = (rows: Array<any>, sortColumnName: string,
...
 
-    let members = columns
-        .map(col => col.valueMember);
-    const newRows = rows.filter(row => {      
+    let members = columns.map(col => col.valueMember);
+
+    const newRows = rows.filter(row => {
         let visible = false;
         for (let value of members) {
-            if (row[value].toString().toLowerCase().includes(searchText)) {
```
Comment (sglogovic) [inline src/components/QuickGrid/DataSelectors.ts]: Why did you replace includes with indexOf?

File: src/components/QuickGrid/QuickGrid.Props.ts (hunk 1 above)
Comment (mpisacic) [reply src/components/QuickGrid/QuickGrid.Props.ts]: yes i moved it in other file

File: src/components/QuickGrid/DataSelectors.ts (hunk 2 above)
Comment (mpisacic) [reply src/components/QuickGrid/DataSelectors.ts]: indexOf is just older function, its just so that both of grids have the same function

**Pull request merged**